content/02.intro.md

```diff
@@ -1,10 +1,10 @@
 ## Introduction
 
-Deep learning (DL), a subfield of machine learning (ML) that focuses on deep artificial neural networks, is exploding in popularity and is increasingly being used for biological data analysis [@doi:10.1098/rsif.2017.0387].
-However, DL itself remains an active area of research, and the complexity of this field poses a large barrier of entry to those who would like to correctly utilize state-of-the-art DL technology in biological research applications.
-While much has been written about the impressive results of DL in biology, there is a comparative dearth of literature articulating best practices for its use, with most instructional literature focusing on ML, rather than DL [@doi:10.1186/s13040-017-0155-3].
+Deep learning (DL), a subfield of machine learning (ML) implementing hierarchical artificial neural networks, is increasingly used for the analysis of biological data [@doi:10.1098/rsif.2017.0387].
+Despite its growing popularity, DL itself remains an active area of research. Its everchanging complexity and lack of current beginner resources focused on biological applications pose large barriers of entry to newcomers who wish to utilize state-of-the-art DL in their research.
+Biological insight garnered from DL has been well-documented in the scientific literature, with applications ranging from predicting protein-drug binding kinetics [@doi:10.1038/s41467-017-02388-1] to identifying the lab-of-origin of synthetic DNA [@doi:10.1038/s41467-018-05378-z]. However, few resources articulate DL best practices to the scientific community. Most instructional literature focuses on ML broadly, rather than DL specifically, further limiting accessibility and reproducibility [@doi:10.1186/s13040-017-0155-3].
 
-To fix this problem, we solicited input from a diverse community of researchers, who wrote this manuscript collaboratively using the GitHub version control platform [@url:https://github.com/Benjamin-Lee/deep-rules] and Manubot [@url:https://greenelab.github.io/meta-review/].
+To address this issue, we solicited input from a diverse community of researchers, who wrote this manuscript collaboratively using the GitHub version control platform [@url:https://github.com/Benjamin-Lee/deep-review] and Manubot [@url:https://greenelab.github.io/meta-review/].
```

Comment:
We shouldn't start a new paragraph with "this issue" without defining "this." On the flipside, we don't want to sound too repetitive. To resolve this, I think this sentence can just be added at the end of the previous paragraph -- would fit nicely there imho

Reply:
Done